content/rs/databases/connect/_index.md

```diff
@@ -13,8 +13,107 @@ aliases: [
 
 ]
 ---
-Learn how to connect your application to a Redis database hosted by Redis Enterprise Software and test your connection.
 
-[Supported connection clients]({{< relref "/rs/databases/connect/supported-clients-browsers.md" >}})
+After you have [Set up a cluster]({{< relref "/rs/clusters/new-cluster-setup" >}}) and [created a Redis database]({{< relref "/rs/databases/create" >}}), connect to your database to test your connection.
 
-[Test client connectivity]({{< relref "/rs/databases/connect/test-client-connectivity.md" >}})
\ No newline at end of file
+To connect to your database, you need the database endpoint, which includes the cluster name (FQDN) and the database port. Select the Configuration tab in the database screen to find the database endpoint.
```

Comment:
```suggestion
To connect to your database, you need the database endpoint, which includes the cluster name (FQDN) and the database port. 

Select the **Configuration** tab in the database screen to find the database endpoint.
```

Reply:
I felt like the action set was a bit buried here; putting it on a new line might help.